libdnf/rpm/reldep.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright (C) 2018-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "libdnf/rpm/reldep.hpp"
+#include "sack_impl.hpp"
+#include "solv/reldep_parser.hpp"
+
+/* workaround, libsolv lacks 'extern "C"' in its header file */
```

Comment:
Style: Do not use multi line comments in code. It is on more places.

Reply:
Fixed